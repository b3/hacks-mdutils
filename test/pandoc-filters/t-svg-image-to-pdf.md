![tux](tux.svg)\ 

![tux](tux.pdf)\ 

![tux](tux.png)\ 

![tux](tux.jpg)\ 
