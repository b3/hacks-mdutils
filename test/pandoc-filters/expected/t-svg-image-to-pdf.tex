\includegraphics{tux.pdf}~

\includegraphics{tux.pdf}~

\includegraphics{tux.png}~

\includegraphics{tux.jpg}~
